components/host-dashboard/Dashboard.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
:eye_speech_bubble: Now that the `Expenses` and Orders part are clearly separated, maybe we should split in separate components?